gnosis/conditional-markets-interface
# Tier 3 request and feedback

# Description
This PR adds the option to request Tier3 and also adds user feedback during the process

# Code
This PR will:

- Add the necessary components to handle Tier3 request

# To Test
- Try to request Tier 3
  - Try all flows of requesting the email again and then moving the user through different states to see feedback given by the app
